alembic/versions/2d52de92d6a9_add_structure_id_to_location_model.py

```diff
@@ -0,0 +1,52 @@
+"""add structure id to location model
+
+Revision ID: 2d52de92d6a9
+Revises: 99ec6a5fa861
+Create Date: 2019-03-04 17:42:54.138722
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2d52de92d6a9'
+down_revision = '99ec6a5fa861'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('structure',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=50), nullable=True),
+    sa.Column('tag_id', sa.Integer(), nullable=True),
+    sa.Column('lft', sa.Integer(), nullable=False),
+    sa.Column('rgt', sa.Integer(), nullable=False),
```

Comment:
Lets have a sync about this migration file because it is currently not reflecting your changes @ThaDeveloper